src/configure

```diff
@@ -184,6 +189,29 @@ do
   --cudatk-dir=*)
     CUDATKDIR=`read_dirname $1`;
     shift ;; #CUDA is used in src/cudamatrix and src/nnet{,bin} only
+  --host=*)
+    # We are going to prepend this to compilation tool invocations, so we need to take
+    # the specified host triple and at a '-'
+    HOST=`expr "X$1" : '[^=]*=\(.*\)'`-;
+    shift ;;
+  --fst-version=*)
```

Comment:
From the commit log for that change:
    When cross compiling, I typically have the built openfst libraries
    installed to an architecture specific directories (without the openfst
    Makefiles).  This switch allows me to specify the openfst version
    without having to keep the Makefile installed beside the libraries.

So I have my android toolchain installed to `/opt/cross/arm-linux-androideabi`.  I have my openfst libraries installed in `/opt/cross/arm-linux-androideabi/lib` and `/opt/cross/arm-linux-androideabi/include` but the Makefile for openfst is not in the root directory.  This switch allows me to specify the openfst version without having to hand copy the Makefile into place with every openfst change (or on any new build host).